dependency-check-maven-config.xml

```diff
@@ -192,4 +192,11 @@
         <cve>CVE-2017-12629</cve>
     </suppress>
 
+    <suppress>
+        <notes>
+            This CVE is related to hash collisions at the Java level. This is mitigated by newer versions of Java.
```

Comment:
could mention JEP-180 and the java version this was fixed in to be more specific.

Reply:
Fixed